src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/DotNetCoreProjectCompatibilityDetector.cs

```diff
@@ -334,57 +344,48 @@ private CompatibilityLevel GetCompatibilityLevelFromVersion(Version version, Ver
         /// less than 24 hours old, it uses that data. Otherwise it downloads from the server. If the download fails it will use the previously cached
         ///  file, or if that file doesn't not exist, it uses the data baked into this class
         /// </summary>
-        private async Task<VersionCompatibilityData> GetVersionCmpatibilityDataAsync()
+        private VersionCompatibilityData GetVersionCmpatibilityData()
```

Comment:
Just noticed a missing "o" in Compatibility...

Reply:
thanks